bigbluebutton/greenlight
# No search results page

## Description

- Adding a no search result page to Server rooms, Manage Users (Active, Pending, Banned, Invited), and Roles
- Refactored existing `NoRoomsFound` and `NoRecordingsFound` components into one component that could be used for all called `NoSearchResults`
## Testing Steps

- Go to those pages and search for something that doesn't exist and make sure no results page shows nicely
## Screenshots (if appropriate):
